### ISSUE TITLE
Enums don't work

### ISSUE THREAD

**Issue by axc** [battlecode/battlecode-server#345]
http://www.battlecodeforum.org/t/problem-with-enums/294/2

**Comment by Pear0** [comment]
The stacktrace contains the class name with an extra semicolon so this may be related to #346 .

**Comment by BrianGuo** [comment]
I get an error when trying to use enums as well, and it's fixed when I remove the enum, but I get a different stack trace:

~~~~~
battlecode.instrumenter.InstrumentationException: ILLEGAL Team is known to have errors
        at battlecode.instrumenter.TeamClassLoaderFactory$Loader.loadClass(TeamClassLoaderFactory.java:388)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at sun.misc.Unsafe.defineClass(Native Method)
        at sun.reflect.ClassDefiner.defineClass(ClassDefiner.java:63)
        at sun.reflect.MethodAccessorGenerator$1.run(MethodAccessorGenerator.java:399)
        at sun.reflect.MethodAccessorGenerator$1.run(MethodAccessorGenerator.java:394)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.reflect.MethodAccessorGenerator.generate(MethodAccessorGenerator.java:393)
        at sun.reflect.MethodAccessorGenerator.generateMethod(MethodAccessorGenerator.java:75)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:53)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at battlecode.instrumenter.SandboxedRobotPlayer.setBytecodeLimit(SandboxedRobotPlayer.java:269)
        at battlecode.world.control.PlayerControlProvider.runRobot(PlayerControlProvider.java:132)
        at battlecode.world.control.TeamControlProvider.runRobot(TeamControlProvider.java:101)
        at battlecode.world.GameWorld.lambda$updateRobots$1(GameWorld.java:132)
        at gnu.trove.map.hash.TIntObjectHashMap.forEachValue(TIntObjectHashMap.java:402)
        at battlecode.world.ObjectInfo.eachRobot(ObjectInfo.java:112)
        at battlecode.world.GameWorld.updateRobots(GameWorld.java:130)
        at battlecode.world.GameWorld.runRound(GameWorld.java:96)
        at battlecode.server.Server.runMatch(Server.java:253)
        at battlecode.server.Server.run(Server.java:167)
        at battlecode.server.Main.runHeadless(Main.java:77)
        at battlecode.server.Main.run(Main.java:109)
        at battlecode.server.Main.main(Main.java:119)
~~~~~

**Comment by axc** [comment]
Was fixed in the process of fixing #346
